app/api/datastore/v1/reporting.rb

```diff
@@ -0,0 +1,34 @@
+module Datastore
+  module V1
+    class Reporting < Base
+      version 'v1', using: :path
+
+      resource :reporting do
+        resource :volumes_by_office do
+          route_setting :authorised_consumers, %w[crime-review]
+
+          params do
+            requires(
+              :period,
+              type: String,
+              year_month_format: true,
+              desc: "Month in '%Y-%B' format (e.g. '2025-November')"
+            )
+
+            optional(
+              :application_types,
+              type: [String],
+              default: Types::ApplicationType.values - [Types::ApplicationType['post_submission_evidence']],
+              values: Types::ApplicationType.values,
+              desc: "Application types to be counted. Defaults to all types except 'Post Submission Evidence'."
```

Comment:
Do we need a spec to check post submission evidence applications will not be included? or will this spec be in review?

Reply:
It is specced here https://github.com/ministryofjustice/laa-criminal-applications-datastore/pull/444/files#diff-3e19ccd2fc5d548f4088dddee5fb0072b5b0289e01817b12c12c6e113bba42a3R55-R57

But I agree not explicitly enough. I'll updated